source/Ship.h

```diff
@@ -157,6 +158,7 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	// Get this ship's cost.
 	int64_t Cost() const;
 	int64_t ChassisCost() const;
+	int64_t LocalCost(const Planet *planet, const ConditionsStore &conditions) const;
```

Comment:
This is another thing that I think should be in that separate class I keep mentioning.